oci/defaults.go

```diff
@@ -154,6 +154,10 @@ func DefaultLinuxSpec() specs.Spec {
 			{Type: "pid"},
 			{Type: "ipc"},
 		},
+		// Always use MS_PRIVATE for rootfs propagation to make sure that we
+		// don't leak any MS_UNBINDABLE mounts into the container's namespace
+		// while its resolving volume bindmounts.
+		RootfsPropagation: "private",
```

Comment:
From my reading of `libcontainer/rootfs_linux.go` this only changes how _the host's_ `/` mount is set before setting up the container's mounts. I'm actually very confused why `MS_SLAVE|MS_REC` is the default, as that seems to me like it would cause a lot of issues.